.github/workflows/sync-version-with-upstream.yml

```diff
@@ -22,6 +22,6 @@ jobs:
         with:
           token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
           update-script: |
-            # Grabbing latest release
-            VERSION=$(curl -sL https://api.github.com/repos/jenkinsci/jenkins/releases |  jq .[].tag_name -r | grep -v rc | sort -r | head -n 1 | cut -d "-" -f2)
+            # Grabbing latest lts release
+            VERSION=$(curl -sL https://api.github.com/repos/jenkinsci/jenkins/releases |  jq .[].tag_name -r | grep -v rc | sort -r | grep "2.440" | head -n1 | cut -d "-" -f2)
             sed -i 's/^\(version: \).*$/\1'"'$VERSION'"'/' snap/snapcraft.yaml 
```

Comment:
You need to delete the `sync-version-with-upstream.yml` from the `lts` branch - scheduled workflows only run the default branch.

You can make a PR to the `candidate` branch to update the workflow there to handle both - see [example](https://github.com/snapcrafters/gimp/blob/candidate/.github/workflows/sync-version-with-upstream.yml) in the `gimp` repo.

Reply:
OK - let me address that some time this week. Perhaps today if I have the time.